nvim/init.lua

```diff
@@ -69,6 +69,142 @@ require('lazy').setup({
     opts = {}
   },
 
+  {
+    "folke/zen-mode.nvim",
+    dependencies = { "folke/twilight.nvim" },
+    event = "VeryLazy",
+  },
+
+  {
+    "tris203/hawtkeys.nvim",
```

Comment:
Isn't telescope find keymaps doing the same thing?

Reply:
Not exactly especially this feature is missing there and is a life saver https://github.com/tris203/hawtkeys.nvim?tab=readme-ov-file#showing-duplicate-keymaps